docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc

```diff
@@ -153,13 +154,13 @@ public class TokenResource {
 This endpoint has ID, access, and refresh tokens injected.
 It returns a `preferred_username` claim from the ID token, a `scope` claim from the access token, and also a refresh token availability status.
 
-Note that you do not have to inject the tokens - it is only required if the endpoint needs to use the ID token to interact with the currently authenticated user or use the access token to access a downstream service on behalf of this user.
+Note that you do not have to inject the tokens - it is only required if the endpoint needs to use the ID token to interact with the currently-authenticated user or use the access token to access a downstream service on behalf of this user.
```

Comment:
```suggestion
Note that you do not have to inject the tokens - it is only required if the endpoint needs to use the ID token to interact with the currently authenticated user or use the access token to access a downstream service on behalf of this user.
```

Introducing `-` reads unusual

Reply:
I added the hyphen to join the words together as both together formed an adjective that modified the word "user". From a grammatical perspective, we would typically have a hyphen in this instance. However, I did some more research into these types of adjectives and both options are acceptable here. Therefore, I will remove in this instance again, per your suggestion.